crates/graphql/src/schema/objects/candymachine.rs

```diff
@@ -0,0 +1,90 @@
+use scalars::{PublicKey, U64};
+use tables::{candy_machine_datas, candy_machines};
+
+use super::prelude::*;
+use crate::schema::scalars::markers::{TokenMint, Unspecified};
+
+pub type CandyMachineColumns = (
+    candy_machines::address,
+    candy_machines::authority,
+    candy_machines::wallet,
+    candy_machines::token_mint,
+    candy_machines::items_redeemed,
+);
+
+pub type CandyMachineDataColumns = (
+    candy_machine_datas::candy_machine_address,
+    candy_machine_datas::uuid,
+    candy_machine_datas::price,
+    candy_machine_datas::symbol,
+    candy_machine_datas::seller_fee_basis_points,
+    candy_machine_datas::max_supply,
+    candy_machine_datas::is_mutable,
+    candy_machine_datas::retain_authority,
+    candy_machine_datas::go_live_date,
+    candy_machine_datas::items_available,
+);
+
+#[derive(Debug, Clone, GraphQLObject)]
+pub struct CandyMachine {
+    pub address: PublicKey<CandyMachine>,
+    pub authority: PublicKey<Unspecified>,
+    pub wallet: PublicKey<Unspecified>,
+    pub token_mint: Option<PublicKey<TokenMint>>,
+    pub items_redeemed: U64,
+
+    pub uuid: String,
+    pub price: U64,
+    pub symbol: String,
+    pub seller_fee_basis_points: i32,
+    pub max_supply: U64,
+    pub is_mutable: bool,
+    pub retain_authority: bool,
+    pub go_live_date: Option<U64>,
+    pub items_available: U64,
+}
+
+impl<'a, 'b> TryFrom<(models::CandyMachine<'a>, models::CandyMachineData<'b>)> for CandyMachine {
+    type Error = std::num::TryFromIntError;
+
+    fn try_from(
```

Comment:
@ray-kast I removed the `CandyMachineJoined` models object, and used a tuple of tuples in the `select` statement instead.